src/system/NewTabs/Tabs.stories.jsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * Internal dependencies
+ */
+import { NewTabs, TabsTrigger, TabsList, TabsContent, Text } from '..';
+
+export default {
+	title: 'NewTabs',
+	component: NewTabs,
+};
+
+export const Default = () => (
+	<NewTabs defaultValue="all">
+		<TabsList aria-label="See all the content">
```

Comment:
Can you make the `aria-label` or a `title` attribute required for this? People might forget to add this attribute. Title is nice, or caption.

Reply:
It's recommended always to have a label for the tabs.

Reply:
@djalmaaraujo fixed this too (it was an issue with the `children` not being passed. We now have the title attribute to use.